fpga_interchange/chipdb.h

```diff
@@ -346,6 +371,7 @@ NPNR_PACKED_STRUCT(struct ChipInfoPOD {
     RelSlice<PackagePOD> packages;
     RelSlice<WireTypePOD> wire_types;
     RelSlice<GlobalCellPOD> global_cells;
+    RelSlice<BelChainPOD> bel_chains;
```

Comment:
chipdb changes should always be bumping the chipdb version, too: https://github.com/YosysHQ/nextpnr/blob/ea45971dcc81983d5c952e36c762b1f922dfe9f7/fpga_interchange/chipdb.h#L37